src/modules/awake/Awake/Awake.csproj

```diff
@@ -43,6 +43,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Win32.Registry" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Windows.CsWin32" Version="0.2.46-beta">
```

Comment:
installer update needed?

Reply:
Not needed. Creates PInvokes that are added to the executable

Reply:
no, it's a build-time dep.